assignment-2/7s.md

```diff
@@ -1,2 +1,293 @@
+> and working to change those that impede cooperation and unity of effort.
+> 
+> • Resources. Identifying needs for both the short and long terms.
+> • Training, exercises, and education. Maintaining readiness, capturing lessons learned, and
+> preparing future leaders.
+
+- Two music streaming tiers: Spotify Free ( a lot advertisements) and Spotify Premium ( paid, remove advertisements, improve audio quality, download available), this is called 'freemium pricing strategy'.
+- Spotify pays artists based on their 'market share' (the number of streams for their songs as a proportion of total songs streamed on the service). They distribute approximately 70% to rights-holders.
+- Other competitors in the market like Grooveshark have been shut down as part of legal settlements, because strategies that did not compensate the music production, Spotify even having some artist criticism, has manage to hold some balance between their activities and the music production industry.
+- Spotify build open and active coalitions among business partners, creating structured channels aiming to ease the use of playlist on many business branches like media, festivals, not-for-profit organisations, audio hardware companies. They also facilitate partnerships at strategic openings for example fitness partners and gift card retailers. 
+
+##### Structure
+
+> Structure is about how the organization is arranged or configured. Many organizations take a
+> functional structure, in which different branches of the organization specialize in particular
+> functions, such as manufacturing, sales, and service. Another common form is a divisional
+> structure, in which divisions are arrayed according to geographical regions, products, or some
+> other factor, and each division contains a full complement of functional elements (e.g., sales,
+> service, R&D).
+
+Spotify headquarters resides in London and its handles, research and development departments are in Stockholm.
+Like many other matrix management organizations, Spotify subdivides the work among people with similar skills, and they are pooled for work assignments, resulting in more than one manager with their own responsibilities and competencies. All this managers are from that point, reporting hierarchical button up to the CTO and finally to the CEO.
+Further literature does not specify aspect like functional elements (e.g., sales, service, R&D), and how they interact with eachother.
+
+##### Systems
+
+> Systems refers to the rules and regulations, standards, and processes for getting things done and managing the activity of the organization, both formal and informal.
+
+How things get done? 
+according to Spotify [12]: 
+- Spotify has a culture based on agile principles, that went down to Scum at early years, but some of the scrum practices were getting on the way so they decided to make them optional. Perhaps because they valorate more the agile principals than the scrum practices.
```

Comment:
"Perhaps because they valorate more the agile principals than the scrum practices." => "Perhaps because they value agile principles more than scrum practices"